frontend/app/src/components/helper/RowAppend.vue

```diff
@@ -1,18 +1,22 @@
 <script setup lang="ts">
-import { type PropType } from 'vue';
-
-const props = defineProps({
-  className: {
-    required: false,
-    type: [String, Object] as PropType<string | Record<string, any>>,
-    default: ''
-  },
-  label: { required: false, type: String, default: '' },
-  labelColspan: { required: false, type: [Number, String], default: 1 },
-  leftPatchColspan: { required: false, type: [Number, String], default: 0 },
-  isMobile: { required: true, type: Boolean },
-  rightPatchColspan: { required: false, type: [Number, String], default: 0 }
-});
+const props = withDefaults(
+  defineProps<{
+    className?: string | Record<string, any>;
+    label?: string;
+    labelColspan?: string | number;
+    leftPatchColspan?: string | number;
+    rightPatchColspan?: string | number;
+    isMobile?: boolean;
```

Comment:
this was required before though

Reply:
`isMobile` yes, but it is something that is related to `vuetify`, and it's `VDataTable` component.

We don't have a mobile version at the moment and I don't think it makes sense to force it.

We will see how we will handle this going forward.